decompiler/extractor/main.cpp

```diff
@@ -421,7 +418,20 @@ int main(int argc, char** argv) {
 
     int flags = 0;
     if (std::filesystem::is_regular_file(data_dir_path)) {
-      // it's a file, treat it as an ISO
+      // it's a file, make sure it's an .iso file
+      //to-do: verify game header data as well
+      if (data_dir_path.extension() != ".iso") {
```

Comment:
Make sure you lowercase the extension (idk if `extension()` does that by default) before comparing in case someone has a file called `JAK1.ISO`.

Reply:
Good catch here because `extension()` is definitely case sensitive lol